base/test.jl

```diff
@@ -138,7 +138,7 @@ type Broken <: Result
     orig_expr
 end
 function Base.show(io::IO, t::Broken)
-    print_with_color(:yellow, io, "Test Broken\n")
+    print_with_color(:yellow, io, "Test Broken\n"; bold = true)
```

Comment:
the other PR that changes warnings to yellow should think about making these use the configurable warning color